main.go

```diff
@@ -47,4 +56,28 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 	if m.Content == "o/" {
 		s.ChannelMessageSend(m.ChannelID, "\\o")
 	}
+
+	if m.Content == "!stoic" {
+		url := "https://stoicquotesapi.com/v1/api/quotes/random"
+
+		resp, err := http.Get(url)
+
+		if err != nil {
+			fmt.Println("error retrieving stoicquotesapi", err)
+			return
+		}
+
+		defer resp.Body.Close()
+
+		var respBody StoicResponse
+		decodeErr := json.NewDecoder(resp.Body).Decode(&respBody)
+
+		if decodeErr != nil {
+			fmt.Println("error decoding stoicquotesapi response", decodeErr)
```

Comment:
```suggestion
		err = json.NewDecoder(resp.Body).Decode(&respBody)

		if err != nil {
			fmt.Println("error decoding stoicquotesapi response", err)
```

Usually `err` is reused.